tests/strands/models/test_litellm.py

```diff
@@ -55,516 +59,44 @@ def test_update_config(model, model_id):
-        "stream_options": {
-            "include_usage": True,
-        },
-        "tools": [],
-    }
-
-    assert tru_request == exp_request
-
-
-def test_format_request_with_params(model, messages, model_id):
-    model.update_config(params={"max_tokens": 1})
-
-    tru_request = model.format_request(messages)
-    exp_request = {
-        "messages": [{"role": "user", "content": [{"type": "text", "text": "test"}]}],
-        "model": model_id,
-        "stream": True,
-        "stream_options": {
-            "include_usage": True,
-        },
-        "tools": [],
-        "max_tokens": 1,
-    }
-
-    assert tru_request == exp_request
-
-
-def test_format_request_with_system_prompt(model, messages, model_id, system_prompt):
-    tru_request = model.format_request(messages, system_prompt=system_prompt)
-    exp_request = {
-        "messages": [
-            {"role": "system", "content": system_prompt},
-            {"role": "user", "content": [{"type": "text", "text": "test"}]},
-        ],
-        "model": model_id,
-        "stream": True,
-        "stream_options": {
-            "include_usage": True,
-        },
-        "tools": [],
-    }
-
-    assert tru_request == exp_request
-
-
-def test_format_request_with_image(model, model_id):
-    messages = [
-        {
-            "role": "user",
-            "content": [
-                {
-                    "image": {
-                        "format": "jpg",
-                        "source": {"bytes": b"base64encodedimage"},
-                    },
-                },
-            ],
-        },
-    ]
-
-    tru_request = model.format_request(messages)
-    exp_request = {
-        "messages": [
-            {
-                "role": "user",
-                "content": [
-                    {
-                        "image_url": {
-                            "detail": "auto",
-                            "format": "image/jpeg",
-                            "url": "data:image/jpeg;base64,base64encodedimage",
-                        },
-                        "type": "image_url",
-                    },
-                ],
-            },
-        ],
-        "model": model_id,
-        "stream": True,
-        "stream_options": {
-            "include_usage": True,
-        },
-        "tools": [],
-    }
-
-    assert tru_request == exp_request
-
-
-def test_format_request_with_reasoning(model, model_id):
-    messages = [
-        {
-            "role": "user",
-            "content": [
-                {
-                    "reasoningContent": {
-                        "reasoningText": {
-                            "signature": "reasoning_signature",
-                            "text": "reasoning_text",
-                        },
-                    },
-                },
-            ],
-        },
-    ]
-
-    tru_request = model.format_request(messages)
-    exp_request = {
-        "messages": [
+@pytest.mark.parametrize(
+    "content, exp_result",
```

Comment:
A description/human-readable name might be useful too to make it more clear what it's testing.

E.g. "thinking transfer", "videos", "plain text"